### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by Poussinou** [rectangle-dbmi/Realtime-Port-Authority#346]
Hi @rectangle-dbmi ,

Since this app is GNU GPL v3 licensed, are you considering the fact to add it to [F-Droid](https://f-droid.org/)? There is a strong community behind.

You can find some documentation [here](https://f-droid.org/docs/).

Otherwise, if you don't want to work on the inclusion of this app in F-Droid by yourself, would you still agree with the inclusion of your app in F-Droid by the maintainer team?

While we're waiting for the inclusion of this app, **could you please attach a signed release apk in your [releases section](https://github.com/rectangle-dbmi/Realtime-Port-Authority/releases), at least for the [last version](https://github.com/rectangle-dbmi/Realtime-Port-Authority/releases/tag/9.0.0)?**

**Comment by epicstar** [comment]
Hello.... I'm slowly trying to get back to PAT Track (currently not employed RIP). I'll add it to F-Droid, but just keep note that I'm using Google Maps. I'm not sure what the terms are with using Google Maps with F-Droid but I'll add this in.

**Comment by Poussinou** [comment]
Using google maps means you included a non FLOSS dependency in your app. Sadly, your app can't be included in F-Droid if you have non-free dependencies in the source.

A workaround might be to attach a signed release apk to your Releases Section to help people get your app outside of Google Play (even if they will still be dependent on Google Maps to use your app). We could try to include it in [IzzyOnDroid repo](https://apt.izzysoft.de/fdroid/) which allows apps with a few proprietary dependencies to be included in. This repo is compatible with a F-Droid client.

Another solution would be to replace Google Maps with OpenStreetMap, but that might be a lot of work.

**Comment by epicstar** [comment]
Yeah, I think the best solution IMO is using OpenStreetMap.... Good news is that the viewmodel doesn't care about the map API. We just have to swap out the View layer depending on the flavor. It's not excessively hard but it still is though :(

**Comment by epicstar** [comment]
We'll try the IzzoOnDroid method. I can easily include a signed APK on the releases page. I'm personally planning to add CD to the Play Store and to the releases page anyway...
